src/test/VoteMonitor.Api.IntegrationTests/Setup/HttpClientExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Net.Http;
+
+namespace VoteMonitor.Api.IntegrationTests.Setup
+{
+    public static class HttpClientExtensions
+    {
+        public static HttpClient WithAuthorizationPolicy(this HttpClient client, Policy policy, params (string type, string value)[] otherClaims)
```

Comment:
Can we create a simple POCO class for claims? Just to improve the readability

Reply:
Done